exotations/exotica_core_task_maps/src/eff_box.cpp

```diff
@@ -105,6 +147,16 @@ void EffBox::Instantiate(const EffBoxInitializer& init)
         eff_lower_[eff_id + 1] = frame.YLim[0];
         eff_lower_[eff_id + 2] = frame.ZLim[0];
     }
+
+    if (debug_ && Server::IsRos())
+    {
+        pub_markers_ = Server::Advertise<visualization_msgs::MarkerArray>("eff_box_objects", 1, true);
+        visualization_msgs::Marker md;  // delete previous markers
+        md.action = visualization_msgs::Marker::DELETEALL;
```

Comment:
This won't compile on older versions, make it  = 3; and add a comment it corresponds to deleteall